content/actions/learn-github-actions/expressions.md

```diff
@@ -55,7 +55,7 @@ As part of an expression, you can use `boolean`, `null`, `number`, or `string` d
 | `boolean` | `true` or `false` |
 | `null`    | `null` |
 | `number`  | Any number format supported by JSON. |
-| `string`  | You don't need to enclose strings in `{% raw %}${{{% endraw %}` and `{% raw %}}}{% endraw %}`. However, if you do, you must use single quotes (`'`) around the string. To use a literal single quote, escape the literal single quote using an additional single quote (`''`). Wrapping with double quotes (`"`) will throw an error. |
+| `string`  | You don't need to enclose strings in `{% raw %}${{{% endraw %} and {% raw %}}}{% endraw %}`. However, if you do, you must use single quotes (`'`) around the string. To use a literal single quote, escape the literal single quote using an additional single quote (`''`). Wrapping with double quotes (`"`) will throw an error. |
```

Comment:
I'm not sure why you made this commit.

https://docs-14895--string-literals.herokuapp.com/en/actions/learn-github-actions/expressions
![image](https://user-images.githubusercontent.com/2119212/152399026-943f5bcb-0bec-4e96-8615-765637254cec.png)

I'd expect the word _and_ to not be in gray....

Reply:
Shouldn't the whole thing be in gray?

Reply:
no, because you're using `${{` to start a thing and `}}` to end it, they're enclosing something else. The `and` isn't part of the raw code, it's descriptive text.

Reply:
Makes sense.